mix.exs

```diff
@@ -53,7 +53,7 @@ defmodule RIG.MixProject do
   def application do
     [
       mod: {Rig.Application, []},
-      extra_applications: [:logger, :runtime_tools, :prometheus_ex, :prometheus_plugs],
+      extra_applications: [:runtime_tools, :prometheus_ex, :prometheus_plugs, :jason, :logger_json],
```

Comment:
is this really necessary? especially for `jason`, which we are already using in the code base without including it here

Reply:
I removed both now. Still works. I had them in because documentation requested it. But if it works, it works I guess 😅

Reply:
So...done :)